weaviate/batch/crud_batch.py

```diff
@@ -583,17 +747,29 @@ class NonExistingClass not present"
         """
 
         if len(self._objects_batch) != 0:
+            warnings.warn(
+                message=BATCH_MANUAL_USE_W,
+                category=RuntimeWarning,
+                stacklevel=1,
+            )
+
             response = self._create_data(
                 data_type='objects',
                 batch_request=self._objects_batch,
             )
-            self._objects_per_second_frame.append(
+            self._objects_batch = ObjectsBatchRequest()
+
+            if response is None:
```

Comment:
This is never going to be `None`, I fixed the `_create_data()` return type. We might as well just remove this block,

Reply:
removed